MonitorControl/Extensions/Preferences+Extension.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Preferences+Extension.swift
+//  MonitorControl
+//
+//  Created by Joni Van Roost on 22/11/2020.
+//  Copyright © 2020 Guillaume Broder. All rights reserved.
```

Comment:
We should remove my name from the "Organization" field in the xcodeproj to remove copyright mentions when generating new files. (And also remove them from current files in the future

Reply:
Done 🙂 The only place I haven't updated is the bundle identifier since I thought that'd probably have an impact on other things.